closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -2394,18 +2427,18 @@ Java_jdk_crypto_jniprovider_NativeCrypto_ECNativeGF2m
 }
 
 static int
-getArrayFromBN(const BIGNUM *bn, unsigned char *out, int len)
+getArrayFromBN(const BIGNUM *bn, unsigned char *out, int off, int len)
```

Comment:
Suggest this change is not necessary: callers can provide `&out[off]` instead of `out` and `off`.

Reply:
I know it is not necessary, but doing it that way makes it way more readable for future use. Is it OK if we keep it as is?

Reply:
I disagree with the "readable" comment. It both complicates understanding of this function and requires (existing) callers to specify an addition argument.

In Java code you must pass an array and an offset separately, but there's no need to split that information into two pieces in C code, especially when the receiver only combines them in a way that the caller could. It also avoids the need to modify those existing call sites.

Reply:
Changed to suggestion.